Assets/Scripts/Bots/BasicBotScript.cs

```diff
@@ -9,9 +9,11 @@ public class BasicBotScript : IBotScript
 	
 	public BasicBotScript() { }
 
-	public BotCommands GetCommands(List<Collider2D> colliders)
-	{
-		Dictionary<String, List<Collider2D>> colliderMap = new();
+	public BotCommands GetCommands(string botinput)
+    {
+        var colliders = new List<Collider2D>();
```

Comment:
Confused as to why we are using collider2ds?

Reply:
I didnt want to break existing code in our bot scripts. 
 
 so we will need to rewrite out bot script using the new input. That line is to be removed.